vm/core/gas.go

```diff
@@ -43,5 +43,5 @@ func TxDataGas(size int) uint64 {
 }
 
 func MaxGas(inputSize int) uint64 {
-	return 10_000 + uint64(inputSize)*150
+	return 4_000 + uint64(inputSize)*100
```

Comment:
Why there is used a (repeated) numeric value instead of using defined above const?